code/obj/machinery/nuclearbomb.dm

```diff
@@ -288,10 +288,10 @@
 
 		if(src.material) src.material.triggerOnBullet(src, src, P)
 
-		if (!damage)
+		if (damage <= 0)
 			return
 		if(P.proj_data.damage_type == D_KINETIC || (P.proj_data.damage_type == D_ENERGY && damage))
-			src.take_damage(damage / 1.7)
+			src.take_damage(max(0.1, damage / 3))
```

Comment:
why max?

Reply:
I wanted a minimum damage value and if I have to use clamp I have to guess an upper bound.

Reply:
I don't believe it will ever actually come up? The only time you would probably do less damage is with a non-damaging projectile. But you also added a check against those above

Reply:
Damage is rounded to the nearest whole number though. At one damage it comes out to 0.3333, at 0 you added a check to return